profile-googlefonts/src/lib.rs

```diff
@@ -174,7 +177,9 @@ include_profiles = ["universal"]
     "googlefonts/STAT/axis_order",
     "googlefonts/STAT/axisregistry",
     "googlefonts/unitsperem",
-    "googlefonts/usweightclass",
+    "googlefonts/weightclass",
+    "googlefonts/varfont/bold_wght_coord",
+    "googlefonts/varfont/duplicate_instance_names",
```

Comment:
`varfont/duplicate_instance_names` now lives at Universal profile.

(https://github.com/simoncozens/fontspector/commit/7cc0e15f42ffbf1d512f2fa50d42fe12ba3aca44)

Reply:
Thanks, I think that was probably rebasing noise.